utils/airtable.js

```diff
@@ -0,0 +1,44 @@
+import findMatchingRecord from './findMatchingRecord';
```

Comment:
One suggestion for the file structure: we probably want the `utils` folder to be inside the `src` folder. Typically we want the project root to be mostly configuration stuff. `src` is where our main application code/business logic should live.

Reply:
Sure, that makes sense! 

Should be changed here: https://github.com/sparkbox/accessible-components/pull/34/commits/6f780decbc1db35ed3c1233dca50a4a9302016d1